src/django_peertube_runner_connector/utils/job_handlers/transcription_job_handler.py

```diff
@@ -0,0 +1,128 @@
+"""Job handler for transcription jobs."""
+import logging
+import uuid
+
+from django.urls import reverse
+
+from django_peertube_runner_connector.models import (
+    RunnerJob,
+    RunnerJobState,
+    RunnerJobType,
+    Video,
+)
+from django_peertube_runner_connector.storage import video_storage
+from django_peertube_runner_connector.utils.files import (
+    generate_transcription_filename,
+    get_video_directory,
+)
+
+from .abstract_job_handler import AbstractJobHandler
+from .utils import (
+    is_transcription_language_valid,
+    load_runner_video,
+    on_transcription_ended,
+)
+
+
+logger = logging.getLogger(__name__)
+
+# https://github.com/Chocobozzz/PeerTube/blob/develop/server/core/lib/runners/job-handlers/transcription-job-handler.ts
+
+
+class TranscriptionJobHandler(AbstractJobHandler):
+    """Handler for transcription jobs."""
+
+    def is_abort_supported(self):
+        """Return True if the job supports aborting."""
+        return True
+
+    def specific_abort(self, runner_job: RunnerJob):
+        """Abort the runner job."""
+
+    def specific_update(self, runner_job: RunnerJob, update_payload=None):
+        """Update the runner job with the payload."""
+
+    def specific_error(
+        self, runner_job: RunnerJob, message: str, next_state: RunnerJobState
+    ):
+        """Handle the error."""
+        if next_state != RunnerJobState.ERRORED:
+            return
+
+        video = load_runner_video(runner_job)
+        if not video:
+            return
+
+        video.decrease_job_info("pendingTranscript")
+
+    def specific_cancel(self, runner_job: RunnerJob):
+        """Cancel the runner job."""
+        video = load_runner_video(runner_job)
+        if not video:
+            return
+
+        video.decrease_job_info("pendingTranscript")
+
+    # pylint: disable=arguments-differ
+    def create(self, video: Video, domain: str):
+        """Create a transcription job."""
+        job_uuid = uuid.uuid4()
+
+        payload = {
+            "input": {
+                "videoFileUrl": domain
+                + reverse(
+                    "runner-jobs-download_video_file",
+                    args=(
+                        video.uuid,
+                        job_uuid,
+                    ),
+                ),
+            },
+        }
+
+        private_payload = {
+            "videoUUID": str(video.uuid),
+        }
+
+        job = self.create_runner_job(
+            domain=domain,
+            job_type=RunnerJobType.VIDEO_TRANSCRIPTION,
+            job_uuid=job_uuid,
+            payload=payload,
+            private_payload=private_payload,
+            priority=0,
+            depends_on_runner_job=None,
+        )
+
+        video.increase_or_create_job_info("pendingTranscript")
```

Comment:
What do you think about creating a constant for this ? or maybe `VideoJobInfoColumnType.PENDING_TRANSCRIPT` ?

Reply:
I already created `VideoJobInfoColumnType.PENDING_TRANSCRIPT` but forgot to use it here